packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -9,720 +9,196 @@ const { generateTemplate } = storybookTemplate('sd-prose');
-    <code>h6</code>.
-  </p>
-  <p>
-    We don't style <code>h6</code> at all out of the box because <code>h5</code> elements are already so small that they
-    are the same size as the body copy. What are we supposed to do with an <code>h5</code>, make it smaller than the
-    body copy? No thanks.
-  </p>
-  <h3>We still need to think about stacked headings though.</h3>
-  <h4>Let's make sure we don't screw that up with <code>h4</code> elements, either.</h4>
-  <p>Phew, with any luck we have styled the headings above this text and they look pretty good.</p>
-  <p>
-    Let's add a closing paragraph here so things end with a decently sized block of text. I can't explain why I want
-    things to end that way but I have to assume it's because I think things will look weird or unbalanced if there is a
-    heading too close to the end of the document.
-  </p>
-  <p>What I've written here is probably long enough, but adding this final sentence can't hurt.</p>`;
-
 export const Default = {
   render: (args: { [k: string]: any }) => {
     return generateTemplate({
       options: {
-        templateContent: html` <div
-          class="sd-container ${args['sd-prose--inverted-attr'] === true
-            ? 'sd-container--variant-primary'
-            : 'sd-container--variant-white'}"
-        >
-          <div class="%CLASSES%">${args['default-slot'] || content}</div>
+        templateContent: html` <div ${args['sd-prose--inverted-attr'] === true ? 'class="bg-primary p-4"' : ''}>
+          <div class="%CLASSES%">${args['default-slot']}</div>
         </div>`
       },
       args
     });
   }
 };
 
-export const Inverted = {
-  render: (args: { [k: string]: any }) => {
-    return generateTemplate({
-      axis: {
-        x: [{ type: 'attribute', name: 'sd-prose--inverted', values: [false, true] }]
-      },
-      options: {
-        templateContent: html` <div class="p-4">
-          <style>
-            .sd-prose--inverted table {
-              opacity: 0;
-            }
-          </style>
-          <div class="%CLASSES%">${args['default-slot'] || content}</div>
-        </div>`,
-        templateBackgrounds: { alternate: 'x', colors: ['white', 'rgb(var(--sd-color-primary, 0 53 142))'] }
-      },
-      args
-    });
-  }
-};
+/**
+ * - `<h1>`: H1, hidden in cms-modules to make sure H1 is only used once on a page
+ * - `<h2>, <h3>, <h4> and <h5>`: H2, H3, H4 and H5
+ * - `<p> and <p> <strong>`: Paragraph and Paragraph bold
+ * - `<sd-leadtext>`: Leadtext
+ * - `<blockquote>`: Quote
+ * - `<hr>`: Divider
+ * - `<a href>`: Inline Link (text only - no icon before/after)
+ * - `<a href> <strong>`: Inline Link bold (text only - no icon before/after)
+ * - `<ul> <li> / <ol> <li>`: List (unordered/ordered)
+ * - `<figure>`: Image (full-width only – no text wrapping)
+ * - `<figcaption>`: Image description
+ * - `<table>`: Table
+ *
+ * ### Spacing
```

Comment:
Can you put "Spacing" has H5 so that is obvious that the samples also belong to "Styling options"?